docs/source/get_started/integrate_commcare.rst

```diff
@@ -83,33 +98,40 @@ verify that the correct rules (with proper URL to your server) have been set up
                 :align: center
 
 To disable the data forwarding rules, you have to open the project settings on your CommCareHQ account and disable them
-from there. Commcare module can only set up the rules, but cannot disable them.
+from there. The Commcare module can only set up the rules, but cannot disable them.
 
         .. image:: img/commcarehq_stop_forwarding.png
                 :scale: 100 %
                 :alt: CommCareHQ data forwarding
                 :align: center
 
+As you can see the provided url :code:`http://demo.motechproject.org/module/commcare/forms/` doesn't have specified configuration.
+So if the Commcare module will receive data then the default configuration will be used. To work with more then one
```

Comment:
So if the Commcare module receives data, the default configuration will be used.